data_steward/tools/run_deid.py

```diff
@@ -228,15 +239,16 @@ def main(raw_args=None):
 
         parameter_list = [
             '--rules',
-            os.path.join(DEID_PATH, 'config', 'ids', 'config.json'),
-            '--private_key', args.private_key, '--table', tablepath, '--action',
-            args.action, '--idataset', args.input_dataset, '--log', LOGS_PATH
+            os.path.join(DEID_PATH, 'config', 'ids',
+                         'config.json'), '--private_key', args.private_key,
+            '--table', tablepath, '--action', args.action, '--idataset',
+            args.input_dataset, '--log', LOGS_PATH, '--odataset', args.odataset
         ]
 
         if args.interactive_mode:
             parameter_list.append('--interactive')
 
-        field_names = [field.get('name') for field in fields_for(table)]
+        field_names = [field.get('name') for field in fields_for(tablepath)]
```

Comment:
@dcampbell-vumc I was wrong.  This line must remain  as `fields_for(table)]`.  This won't work correctly without this information.